hoomd/pytest/test_filter.py

```diff
@@ -35,7 +35,12 @@ def type_indices(request):
     return deepcopy(request.param)
 
 
-def test_type_filter(make_filter_snapshot, simulation_factory, type_indices):
+def test_type_filter(make_filter_snapshot,
```

Comment:
Please use the `@pytest.mark.serial` function decorator.

Reply:
Fixed 4b2543522b5b98d76daaedabafd12a63b8401bfb